Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -100,18 +119,30 @@ extension HTTPClient {
         /// header is set with the given `length`.
         ///     - stream: Body chunk provider.
         @preconcurrency
+        @available(*, deprecated, message: "Use `stream` with `length: Int64` instead")
```

Comment:
We don't need to use messages here, we can use `renamed` and write out the new function name:

```suggestion
        @available(*, deprecated, renamed: "stream(contentLength:bodyStream:)")
```